src/test/js/multibranch/testResults.js

```diff
@@ -40,8 +40,11 @@ module.exports = {
         blueRunDetailsPage.clickTab('tests');
 
         // Expand the test.
-        browser.useXpath().click('//div[@class="result-item-head"]');
-
-        browser.useXpath().waitForElementVisible('//div[@class="test-console"]/h4')
+        browser.useXpath().waitForElementVisible('//span[contains(text(), "StdOut")]');
```

Comment:
@imeredith please see https://github.com/jenkinsci/blueocean-acceptance-test/pull/88 and https://github.com/jenkinsci/blueocean-acceptance-test/pull/88/files#diff-fbb1a6d3d9df1b771d890e368f081658R443 

You should really drop the usage of useXpath and instead declare those paths in a page object (e.g. in runDetailsPage).